docs/user-guide/creating-application/workflow/cd-pipeline.md

```diff
@@ -106,7 +119,7 @@ Devtron's tool has 4 types of deployment strategies. Click on `Add Deployment st
 
 \(d\) Rolling
 
-### 5. Post-deployment Stage
+### 6. Post-deployment Stage
 
 If you want to Configure actions like Jira ticket close, that you want to run after the deployment, you can configure such actions in the post-deployment stages.
```

Comment:
If you want to run actions like closure of Jira ticket after the deployment, you can configure such actions in the post-deployment stages.